AmplifyPlugins/Auth/Tests/AuthHostApp/AuthIntegrationTests/AuthUserAttributesTests/AuthUserAttributesTests.swift

```diff
@@ -100,6 +100,23 @@ class AuthUserAttributesTests: AWSAuthBaseTest {
             }
         }
         wait(for: [updateExpectation], timeout: networkTimeout)
+        
+        let fetchUserAttributeExpectation = expectation(description: "Fetch User Attribute operation should complete")
+        
+        _ = Amplify.Auth.fetchUserAttributes(listener: { result in
+            switch result {
+            case .success(let attributes):
+                if let emailAttribute = attributes.filter({ $0.key == .email }).first {
+                    XCTAssertEqual(emailAttribute.value, updatedEmail)
```

Comment:
Does this work? We did not verify the attribute correct?

Reply:
Yes. So if you check the entire test `func testSuccessfulUpdateEmailAttribute() throws`.
* it first sign up and sign in the user. 
* Updates the email 
* Fetches attributes to check if email was updated. (Added this additional step to reinforce the test)